lib/ansible/modules/web_infrastructure/sentry_project.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Mikhail Naletov <okgolove@markeloff.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: sentry_project
+short_description: Manage Sentry project
+description:
+    - Manage the state of a project in Sentry.
+version_added: "2.8"
+
+options:
+  api_token:
+    description:
+      - API token generated in Sentry. The token sufficient access to manipulate projects.
+    type: str
+    required: true
+
+  organization:
+    description:
+      - Organization containing a project.
+    type: str
+    required: true
+
+  project_name:
+    description:
+      - Project name you want to work with.
+    type: str
+    required: true
+
+  project_slug:
+    description:
+      - Slug name. Slug name must be unique (even after project deletion you can't create a project with same slug name).
+    type: str
+    required: true
+
+  state:
+    description:
+      - The state of project.
+    type: str
+    required: true
+    choices: [ "present", "absent" ]
+
+  team:
+    description:
+      - The name of the team to create a new project for.
+    type: str
+    required: true
+
+  url:
+    description:
+      - Sentry URL.
+    type: str
+    required: false
+    default: sentry.io
+
+notes:
+  - Module supports check_mode, but it can't provide 100 percent guarantee that the specified slug hasn't been used before.
+author: "Mikhail Naletov (@okgolove)"
+'''
+
+EXAMPLES = '''
+# Create project "backend" in "example" organization and "senior" team
```

Comment:
```suggestion
- name: Create project backend in example organization and senior team
  sentry_project:
```
And add an example with state absent

Reply:
This is strong recommended (to use -name:, no comments)

Reply:
Usage of -name: instead of comments for describinf examples is strong recommended. See the development documentation